test/interactions/dragInteractionTests.ts

```diff
@@ -156,6 +145,49 @@ describe("Interactions", () => {
       assert.deepEqual(receivedStart, startPoint, "was passed the correct starting point");
       assert.deepEqual(receivedEnd, endPoint, "was passed the correct current point");
 
+      assert.strictEqual(drag.onDragEnd(), endCallback, "retrieves the callback if called with no arguments");
+      drag.onDragEnd(null);
+      assert.isNull(drag.onDragEnd(), "removes the callback if called with null");
+      svg.remove();
+    });
+
+    it("constrain()", () => {
```

Comment:
We are definitely going to need to revisit our test structure, because I can see why this test is huge and judging from how our tests are set up, it actually makes sense in terms of our infrastructure... which is not a good thing.